pkg/router/template/template_helper_test.go

```diff
@@ -1061,7 +1112,7 @@ func TestParseIPList(t *testing.T) {
 				}
 				return
 			}
-			if got != tc.input {
+			if got != tc.expectedReturn {
```

Comment:
Followup to https://github.com/openshift/router/pull/621/files#r1815351390.
```suggestion
			if expectedEmpty && got != "" || got != tc.expectedReturn {
```

Reply:
Looks like this was taken care of.